src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -479,15 +474,17 @@ private TypeSymbol BindTupleType(TupleTypeSyntax syntax, DiagnosticBag diagnosti
                 return new ExtendedErrorTypeSymbol(this.Compilation.Assembly.GlobalNamespace, LookupResultKind.NotCreatable, diagnostics.Add(ErrorCode.ERR_TupleTooFewElements, syntax.Location));
             }
 
-            return TupleTypeSymbol.Create(syntax.Location,
-                                            typesArray,
-                                            locationsArray,
-                                            elementNames == null ?
-                                                default(ImmutableArray<string>) :
-                                                elementNames.ToImmutableAndFree(),
-                                            this.Compilation,
-                                            syntax,
-                                            diagnostics);
+            return TupleTypeSymbol.Create(
+                                    syntax.Location,
+                                    typesArray,
+                                    locationsArray,
+                                    elementNames == null ?
+                                        default(ImmutableArray<string>) :
+                                        elementNames.ToImmutableAndFree(),
+                                    this.Compilation,
+                                    syntax,
+                                    diagnostics,
+                                    this.ShouldCheckConstraints);
```

Comment:
Aleksey had a comment earlier (which is no longer visible in Git) to test both the cases of`ShouldCheckConstraints`. Which test has it `true` and which has it `false`?